src/core/jwt/jwtTokenHandler.js

```diff
@@ -54,14 +55,20 @@ const jwtTokenHandlerFactory = function jwtTokenHandlerFactory({serviceName = 't
         if (!refreshToken) {
             throw new Error('Refresh token is not available');
         } else {
-            return coreRequest({
-                url: refreshTokenUrl,
+            return fetch(refreshTokenUrl, {
                 method: 'POST',
-                data: JSON.stringify({ refreshToken }),
-                dataType: 'json',
-                contentType: 'application/json',
-                noToken: true
-            }).then(({ accessToken }) => tokenStorage.setAccessToken(accessToken).then(() => accessToken));
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify({ refreshToken })
+            })
+                .then(response => {
+                    if (response.status === 200) {
+                        return response.json();
+                    }
+                    return Promise.reject(response);
```

Comment:
By convention promises should reject an `Error`, the response can be contained within the error if needed